src/components/device/ace.ts

```diff
@@ -280,6 +281,35 @@ export default class Ace implements IIpDeviceManager, IUVCControls {
     });
   }
 
+  _setCnnFeature(cnnFeature: huddly.CnnFeature): Promise<huddly.DeviceStatus> {
+    return new Promise(async (resolve, reject) => {
+      this.grpcClient.setCnnFeature(cnnFeature, (err, status: huddly.DeviceStatus) => {
+        if (err != undefined) {
+          this.handleError('Unable to set cnn feature', err, reject);
+        }
+        resolve(status);
+      });
+    });
+  }
+
+  startCnnFeature(
```

Comment:
If we decide to keep this start/stop cnnFeature structure, I think the other cnnfeatureControls need to be refactored a bit. Personally I would just like to change these two methods to not be generic, i.e. startDetector()/stopDetector().

Reply:
The most important thing is that we remain consistent with what method is called to do what. If this remains, then Boxfish.ts should implement these methods as well. Maybe best to add these methods as part of a new interface called `ISmartDevice.ts` or somethiing similar. This interface would describe the smart features of a huddly camera which then are implemented in Ace.ts and Boxfish.ts